doc/source/api/solver/settings.rst

```diff
@@ -107,18 +107,21 @@ and ``NamedObject`` types, the state value is a dictionary. For the
 You can also access the state of an object with the ``get_state`` method and
 modify it with the ``set_state`` method.
 
-Real parameter objects can have units as well as values. If a particular object
-supports units then you can obtain its value and units as an ``ansys.units.Quantity``
-object by calling the ``as_quantity`` method. You can obtain the same value 
-and units information as a tuple by calling the ``value_with_units`` method.
-That is especially useful if you do not wish to install the ansys.units package.
-Both ``ansys.units.Quantity`` objects and value-unit tuples can be passed to the
-``set_state`` method.
+The ``Real`` and ``RealList`` parameter objects in PyFluent can incorporate units
+alongside values. If an object supports units, you can retrieve its value and units
+as an ``ansys.units.Quantity`` object using the ``as_quantity`` method. Alternatively,
+you can obtain the same information as a tuple by calling the ``state_with_units``
+method. If the container object supports units, calling ``state_with_units`` on it
+returns a ``dict`` where relevant values are represented as tuples containing both
+the value and units.
+
+Both ``ansys.units.Quantity`` objects and value-unit tuples can be used with the
+``set_state`` method of ``Real`` or ``RealList`` objects.
```

Comment:
```suggestion
The ``Real`` and ``RealList`` parameter objects in PyFluent can incorporate units
alongside values. If an object supports units, you can retrieve its value and units
as an ``ansys.units.Quantity`` object using the ``as_quantity()`` method. Alternatively,
you can obtain the same information as a tuple by calling the ``state_with_units()``
method. If the container object supports units, calling the ``state_with_units()`` method on it
returns a dictionary where relevant values are represented as tuples containing both
the value and units.

Both ``ansys.units.Quantity`` objects and value-unit tuples can be used with the
``set_state()`` method of ``Real`` or ``RealList`` objects.
```
Minor edits as method names are supposed to include the ``()`` after them. Nice working on meeting all other style objectives!

Reply:
Thank you, @PipKat: I had the same thought, and instead followed the approach previously adopted in this file. I will accept this suggested change and update other instances herein.